web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -1,25 +1,167 @@
 ---
+id: "creating-new-drupal-project"
+title: "Creating a New Drupal Project"
+slug: "/Backend Starters/Decoupled Drupal/Creating a New Project"
 sidebar_position: 2
 ---
 
-# Creating a New Project
+## Choosing an Approach
 
-## Prerequisites
+### Use Build Tools if:
+
+- Testing is an important part of your workflow
+
+- You don’t want to manually push changes to your code repo.
+
+### Use Dashboard Upstream if:
+
+- You require a simpler setup
+
+- Pantheon repo is your source of truth
```

Comment:
This still doesn't read as a complete statement to me. Also does the style guide allow 'repo' our should we be using 'repository'